rds/sql/dialect/postgreSQL.ts

```diff
@@ -187,6 +187,11 @@ export abstract class AbstractPostgreSqlDialect extends diaImpl.ANSI {
         ${fn.bodyCode}
         $BODY$;
         -- ALTER FUNCTION ${objDefnName}${argTypes} OWNER TO (TODO: OWNER);
+        INSERT INTO plpgsql_check_result
+        (function_id, line_no, "statement", sql_state, message, detail, hint,
+         "level", "position", query, context)
```

Comment:
@Kunjahamed-P  @divyaSkumarCI01188 It looks like this generated code may not have been tested? Without a semicolon at the end of the insert, the SQL will likely break. Please correct or explain why no semicolon is required at the end.

Reply:
We have already added [semicolon at the end of the insert](https://github.com/gov-suite/governed-im-transform/blob/master/rds/sql/dialect/postgreSQL.ts#L194) .